pkg/scheduler/scheduler.go

```diff
@@ -124,6 +154,9 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		log := log.WithValues("workload", klog.KObj(e.Obj), "clusterQueue", klog.KRef("", e.ClusterQueue))
 		if err := s.admit(ctrl.LoggerInto(ctx, log), e); err != nil {
 			e.inadmissibleMsg = fmt.Sprintf("Failed to admit workload: %v", err)
+		} else if s.waitForPodsReady {
+			klog.InfoS("Workload successfully admitted when waitForPodsReady - do admit more in this cycle", "workload", e.Obj.Name)
```

Comment:
halting more admissions in this cycle.

Admit should already log the name of the workload, so I don't think we should include it again.

Reply:
That's right, cleaned up